src/Resources/app/storefront/src/scss/category-filter.scss

```diff
@@ -39,4 +39,8 @@
 .filter-multi-select-cat > div {
   overflow-x: hidden;
   overflow-y: auto;
-}
\ No newline at end of file
+}
+
+.fl-disabled {
```

Comment:
This fix solves the multiple opacities, but now also active filters and filters that are still available will also have this opacity:

![image](https://user-images.githubusercontent.com/20555112/108682946-40522e00-74f1-11eb-8b88-fb6296fb9d2f.png)